gcloud/bigtable/table.py

```diff
@@ -145,3 +147,41 @@ def delete(self):
         client = self._cluster._client
         # We expect a `._generated.empty_pb2.Empty`
         client._table_stub.DeleteTable(request_pb, client.timeout_seconds)
+
+    def sample_row_keys(self):
+        """Read a sample of row keys in the table.
+
+        The returned row keys will delimit contiguous sections of the table of
+        approximately equal size, which can be used to break up the data for
+        distributed tasks like mapreduces.
```

Comment:
The whole idea is that the sample is used to gauge how much stuff is in the table and not much else.

> returned row keys will delimit contiguous sections of the table of approximately equal size ([ref][1], [ref][2])

[1]: https://github.com/GoogleCloudPlatform/cloud-bigtable-client/blob/6652d799b564eb8138ebac6a1003371a57fb359b/bigtable-protos/src/main/proto/google/bigtable/v1/bigtable_service.proto#L40-L43
[2]: https://github.com/GoogleCloudPlatform/cloud-bigtable-client/blob/6652d799b564eb8138ebac6a1003371a57fb359b/bigtable-protos/src/main/proto/google/bigtable/v1/bigtable_service_messages.proto#L97-L103